packages/manager/apps/web-ongoing-operations/src/pages/upload/Upload.spec.tsx

```diff
@@ -20,21 +16,40 @@ vi.mock('react-router-dom', () => ({
   useParams: () => {
     return {
       id: '1',
-      type: 'corporationProof',
     };
   },
 }));
 
+vi.mock('@/hooks/data/data', () => ({
```

Comment:
```suggestion
vi.mock('@/hooks/data', () => ({
```
maybe you need add index file in data folder for avoid duplication in path